src/v2/Apps/Auction/AuctionApp.tsx

```diff
@@ -29,14 +29,14 @@ export const AuctionApp: React.FC<AuctionAppProps> = ({
 }) => {
   const { contextPageOwnerType, contextPageOwnerSlug } = useAnalyticsContext()
 
+  const showLotStandings = me?.showLotStandingsTab?.length && !sale.isClosed
+
   const tabBar = {
     isVisible:
-      sale.showAssociatedSale ||
-      me?.showLotStandingsTab?.length ||
```

Comment:
If we're no longer using this aliased field, it can be removed from the fragment query